src/components/ListDetail/ListDetail.module.css

```diff
@@ -200,32 +200,11 @@
 * 1) Used so the transparent icon doesn't show the line behind. 
 */
 .list-detail__icon {
+  --icon-size: var(--eds-size-3);
```

Comment:
kept here b/c adding the `size` prop 3 different times in `ListDetail` felt unfortunate. but could add it there for consistency